app/controllers/users_controller.rb

```diff
@@ -1,3 +1,25 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+   @user = User.new
+  end
+
+  def show
+    @user = User.find_by(id: params[:id])
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render :new
```

Comment:
thất bại vẫn phải in ra thông báo nha

Reply:
anh ơi nếu lỗi mà render :new thì tự nó sẽ báo những lỗi trong validate r anh ạ